conf/application.conf

```diff
@@ -174,6 +174,22 @@ microservice {
         context = /declarations/querystatusinformation/v1
       }
     }
+  
+    v2 {
```

Comment:
Can we add a dummy client id to make sure we don't accidentally make this visible?